workers/ohsome_quality_analyst/cli.py

```diff
@@ -114,7 +118,8 @@ def create_indicator(
     fid_field: str,
     force: bool,
 ):
-    """Create an Indicator and print results to stdout."""
+    """Create an Indicator and print results to stdout. Write a GeoJSON if an outfile
+    is specified or an infile is used as input"""
```

Comment:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. 

```suggestion
    """Create an Indicator.
    
    Write a GeoJSON with the result to disk if an outfile is specified.
    Otherwise print to stdout.
    """
```